lib/rubocop/inflector/extensions/rspec.rb

```diff
@@ -12,10 +12,21 @@ def camel_to_snake_case(string)
                 ActiveSupport::Inflector.underscore(string)
               end
             end
+
+            module SpecFilePathFormat
+              def camel_to_snake_case(string)
+                ActiveSupport::Inflector.underscore(string)
+              end
+            end
           end
         end
 
-        ::RuboCop::Cop::RSpec::FilePath.prepend Cop::RSpec::FilePath
+        if const_defined? '::RuboCop::Cop::RSpec::FilePath'
+          ::RuboCop::Cop::RSpec::FilePath.prepend Cop::RSpec::FilePath
+        end
+        if const_defined? '::RuboCop::Cop::RSpec::SpecFilePathFormat'
```

Comment:
Note: I can remove this line if I can update the dependencies declared in gemspec.

Which do you like?
```diff
  # rubocop-inflector.gemspec
-   spec.required_ruby_version = '>= 2.3'
-   spec.add_dependency 'rubocop-rspec'
+   spec.required_ruby_version = '>= 2.7'
+   spec.add_dependency 'rubocop-rspec', '>= 2.24.0'
```

(Ruby 2.7 is required by RuboCop RSpec v2.24.0)